content/chapter-03/AzureCloudStorage.md

```diff
@@ -0,0 +1,18 @@
+# Storage space and IO speed
+
+## [Azure Cloud Storage](https://docs.microsoft.com/en-us/azure/storage/common/storage-introduction)
+
+* [Disks](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview)
+  * Aims Virtual Machine deployment
+  * Two options: [Standard](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/standard-storage) and [Premium](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/premium-storage)
+  
+  
+* [Blob storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+  * Common uses include: Serving images, storing files, streaming video and audio
+  * Three variations: Block blobs, Append blobs, and page blobs
```

Comment:
href links would be nice for each variation

Reply:
References added.